projects/core/src/auth/store/reducers/user-token.reducer.ts

```diff
@@ -34,6 +32,18 @@ export function reducer(
         ...state,
       };
     }
+
+    case fromUpdateEmailAction.UPDATE_EMAIL_SUCCESS: {
+      const updatedUserId: UserToken = {
+        ...state,
+        ...action.newUserId,
+      };
+
+      return {
+        ...updatedUserId,
+        userId: `${updatedUserId.userId}`,
```

Comment:
I guess `newUserId` should be a string; so, no need to convert it to string here